SQL/Archive/17.1/2017-03-24-examiner_decoupling_from_site.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS `examiners_psc_rel`;
```

Comment:
I don't think patches that are intended to run on production should have `DROP TABLE IF EXISTS`.. it's a recipe for accidental data loss.